py-rse/regexp.Rmd

```diff
@@ -0,0 +1,13 @@
+# Regular Expressions {#py-rse-regexp}
```

Comment:
Should we resolve #71 before adding this placeholder? (i.e. `regexp.Rmd` might not be needed)

Reply:
Removed placeholder.